src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -6954,9 +6976,21 @@ namespace ZoneTempPredictorCorrector {
                     }
                 }
             }
+            ZoneTempOscillateDuringOccupancy(iZone) = 0.0;
+            ZoneTempOscillateInDeadband(iZone) = 0.0;
             if (isOscillate) {
                 ZoneTempOscillate(iZone) = TimeStepSys;
                 isAnyZoneOscillating = true;
+                if (allocated(ThermalComfortInASH55)) {
```

Comment:
So, it looks like all of these oscillate gymnastics are always done - even if the oscillating output variables are not requested. Is that true? Seems we could track if any of these have been requested and set a flag to skip all of this.